### PR TITLE
(maint) Changelog for release 4.0.33 and version bump

### PR DESCRIPTION
**Pull request by sebastian-miclea**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+## [4.0.33](https://github.com/puppetlabs/facter/tree/4.0.33) (2020-08-05)
+
+[Full Changelog](https://github.com/puppetlabs/facter/compare/4.0.32...4.0.33)
+
+### Added
+- \(FACT-2040\) Added solaris memory resolver [\#2040](https://github.com/puppetlabs/facter/pull/2040) ([sebastian-miclea](https://github.com/sebastian-miclea))
+
+### Fixed
+- \(FACT-2735\) When no hypervisor is detected, return nil [\#2004](https://github.com/puppetlabs/facter/pull/2004) ([IrimieBogdan](https://github.com/IrimieBogdan))
```
Comment (IrimieBogdan) [inline CHANGELOG.md:9]: Should be the PR title not the commit message

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+- \(FACT-2736\) networking facts don't work on EXADATA baremetal [\#2008](https://github.com/puppetlabs/facter/pull/2008) ([IrimieBogdan](https://github.com/IrimieBogdan))
+- \(FACT-2724\) If no hypervisors is detected, return 'physical' [\#2010](https://github.com/puppetlabs/facter/pull/2010) ([IrimieBogdan](https://github.com/IrimieBogdan))
```
Comment (IrimieBogdan) [inline CHANGELOG.md:11]: Should be the PR title not the commit message

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
+## [4.0.33](https://github.com/puppetlabs/facter/tree/4.0.33) (2020-08-05)
+
+[Full Changelog](https://github.com/puppetlabs/facter/compare/4.0.32...4.0.33)
+
+### Added
+- \(FACT-2040\) Added solaris memory resolver [\#2040](https://github.com/puppetlabs/facter/pull/2040) ([sebastian-miclea](https://github.com/sebastian-miclea))
```
Comment (IrimieBogdan) [inline CHANGELOG.md:6]: The correct link is `https://github.com/puppetlabs/facter/pull/1999`

Comment (sebastian-miclea) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 1 above)
Comment (sebastian-miclea) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (sebastian-miclea) [reply CHANGELOG.md]: Fixed

**Comment by sebastian-miclea** [comment]
jenkins please test this on all

**Comment by sebastian-miclea** [comment]
jenkins please test this on redhat7-64a,ubuntu1604-64a,aix61-POWERfa,aix71-POWERfa,aix72-POWERfa,amazon6-64a,amazon7-64a,debian9-64a,debian9-32a,debian10-64a,fedora30-64a,fedora31-64a,fedora32-64a,osx1014-64a,osx1015-64a,centos6-64a,centos6-32a,centos8-64a,redhat7-POWERa,redhat7-AARCH64a,redhat8-64a,redhatfips7-64a,sles12-64a,sles12-POWERa,sles15-64a,solaris11-64a,solaris114-64a,ubuntu1604-32a,ubuntu1604-POWERa,ubuntu1804-64a,ubuntu2004-64a,windows10ent-64a,windows10ent-32a,windows2012r2-64a,windows2016-64a,windows2019-64a,windowsfips2012r2-64a

**Comment by sebastian-miclea** [comment]
jenkins please test this on redhat7-64a,ubuntu1604-64a,aix61-POWERfa,amazon6-64a,amazon7-64a,debian9-64a,debian9-32a,debian10-64a,fedora30-64a,fedora31-64a,fedora32-64a,osx1014-64a,osx1015-64a,centos6-64a,centos6-32a,centos8-64a,redhat7-AARCH64a,redhat8-64a,redhatfips7-64a,sles12-64a,sles12-POWERa,sles15-64a,solaris11-64a,solaris114-64a,ubuntu1604-32a,ubuntu1604-POWERa,ubuntu1804-64a,ubuntu2004-64a,windows10ent-64a,windows10ent-32a,windows2012r2-64a,windows2016-64a,windows2019-64a,windowsfips2012r2-64a

**Pull request merged**